src/create_disk.sh

```diff
@@ -172,13 +172,17 @@ case "$arch" in
         ;;
     s390x)
         sgdisk_args=()
+        rootpn_length=0
         if [[ ${secure_execution} -eq 1 ]]; then
             # shellcheck disable=SC2206
             sgdisk_args+=(-n ${SDPART}:0:+200M -c ${SDPART}:se -t ${SDPART}:0FC63DAF-8483-4772-8E79-3D69D8477DE4)
+            # we need to leave space for the verity hash partitions
+            rootpn_length=-$((128+256))M
```

Comment:
the name of this var is a bit misleading.. this value isn't the length of the rootpartition. 
```suggestion
            secex_partitions_length=-$((128+256))M
```

Reply:
It's the `sgdisk` length argument of the root partition. But the `n` there is confusing since `rootpn` stands for root partition number, so I renamed it to `rootp_end`.